sys/usb/usbus/cdc/acm/Kconfig

```diff
@@ -49,6 +49,7 @@ endif # KCONFIG_USEMODULE_USBUS_CDC_ACM
 config MODULE_USBUS_CDC_ACM
     bool "USB CDC ACM support"
     depends on MODULE_USBUS
+    default y if TEST_KCONFIG && HAS_HIGHLEVEL_STDIO
```

Comment:
Also don't need `TEST_KCONFIG` here as it is already in `MODULE_USBUS`. Maybe here would be better to `select` though @leandrolanzieri will probably disagree (or maybe imply or something).